2_process.R

```diff
@@ -85,20 +87,33 @@ p2_targets_list <- list(
     write_to_csv(p2_daily_with_seg_ids, "2_process/out/daily_do_data.csv"),
     format = "file"
   ),
-  
-  # Estimate daily (normalized) max-light
-  tar_target(
-    p2_daily_max_light,
-    calc_seg_light_ratio(p1_reaches_sf, start_date = earliest_date, end_date = dummy_date),
-    pattern = map(p1_reaches_sf)
-  ),
 
   # make list of "well-observed" sites
   tar_target(
    p2_well_observed_sites,
    p2_sites_w_segs %>% filter(count_days_total > 300) %>% pull(site_id)
  ),
  
+ # filter p1_reaches_sf to segments with "well-observed" sites
+ tar_target(   
+   p2_well_obs_reaches,
```

Comment:
```suggestion
   p2_well_observed_reaches,
```

Naming suggestion to be consistent with `p2_well_observed_sites` above.

Reply:
If you opt to make this change, `p2_well_obs_reaches` should also be edited below (lines 112 and 114).